src/docs/templates/our-story.hbs

```diff
@@ -0,0 +1,116 @@
+		<p>Hiking up a volcano at dawn one day, an idea took root. What if she could bottle that Bali magic, with beauty products that combined luscious tropical ingredients with serious science?</p>
+		<p>Doing some digging, Emily discovered Bali really is Eden for beauty lovers, with an endless bounty of skin-and-hair-loving fruits and plants. The jewel in the crown? Raw virgin coconuts – with Indonesia being the world’s largest producer of this nourishing, revitalising superfruit.</p>
+		<p>With the final seed planted and the hero ingredient chosen, Coco & Eve was born!</p>
+		<figure class="image-with-text mobile-wrapper row position-relative align-items-center mb-lg-2 mt-5">
+			<div class="col-12 col-lg-8 order-lg-last">
+				<div class="no-gutters__in-container">
+					<picture>
+						<source srcset="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/AboutUs_banner03_DT_2x_dcddf728-2bc3-4e2b-9f01-4fb0b3366751_400x.jpg?v=1590061764" media="(min-width: 992px)" />
+						<img src="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/AboutUs_banner03_DT_2x_1000x_a2009ba0-e16e-404e-9e49-f2c1572890ab.jpg?v=1618403172" class="w-100">
+					</picture>
+				</div>
+			</div>
+			<figcaption class="col-12 col-lg-4">
+				<div class="image-with-text__text-content text-left mt-lg-0 p-2 pt-lg-3 pb-lg-3 pr-lg-3 ">
+					<h1>We've spent years testing, developing and perfecting our formulas</h1>
+					<p>We’re passionate about our customers, so we never use drying and damaging sulfates, phthalates or parabens, (unlike 98.9% of beauty products!). We also love our furry friends, so Coco & Eve is proudly 100% vegan and cruelty-free.</p>
+				</div>
+			</figcaption>
+		</figure>
+		<figure class="image-with-text mobile-wrapper row flex-row-reverse position-relative align-items-center mb-lg-2 mt-5">
+			<div class="col-12 col-lg-8 order-lg-last">
+				<div class="no-gutters__in-container">
+					<picture>
+						<source srcset="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/AboutUs_banner04_DT_2x_6b055d51-178d-47e5-a6b1-c273f08a76aa_800x.jpg?v=1590061796" media="(min-width: 992px)" />
+						<img src="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/AboutUs_banner04_DT_2x_1000x_74c1ad4b-cb44-4123-a152-12d7495fbca4.jpg?v=1618404879" class="w-100">
+					</picture>
+				</div>
+			</div>
+			<figcaption class="col-12 col-lg-4">
+				<div class="image-with-text__text-content image-with-text__text-content--reverse text-left mt-lg-0 p-2 pt-lg-3 pb-lg-3 pl-lg-3">
+					<h1>We've spent years testing, developing and perfecting our formulas</h1>
+					<p>We’re passionate about our customers, so we never use drying and damaging sulfates, phthalates or parabens, (unlike 98.9% of beauty products!). We also love our furry friends, so Coco & Eve is proudly 100% vegan and cruelty-free.</p>
+				</div>
+			</figcaption>
+	</div>
+	<div class="page-award-slider text-center pt-5 pb-5">
+		<div class="container ">
```

Comment:
we don't need to give the width to the P, also you are using a CONTAINER but without really using its features. So let's use the grid and col (maybe 3 cols size?) with centred col.